Cargo.toml

```diff
@@ -8,7 +8,7 @@ readme = "README.md"
 repository = "https://github.com/primait/bridge.rs"
 version = "0.25.0"
 # See https://github.com/rust-lang/rust/issues/107557
```

Comment:
Since we are changing `rust-version` I think we can remove the comment above. It was introduced bu @oliverbrowneprima  in this [commit](https://github.com/primait/bridge.rs/commit/6a3c86b38ca7edb5f92a38727b1d44400d5c5ddb) but I think it was related related to version `1.72`.